templates/webserver/webserver-service.yaml

```diff
@@ -25,3 +25,10 @@ spec:
       protocol: TCP
       port: {{ .Values.ports.airflowUI }}
       targetPort: {{ .Values.ports.airflowUI }}
```

Comment:
We probably don't want the webserver port here and proxy to `localhost:{{ .Values.ports.airflowUI }}` instead of back through the service, right?

Reply:
@pgvishnuram ☝️